README.md

```diff
@@ -1,18 +1,40 @@
 # iBanks NVIDIA powered GPU cluster
 
-## Nodes and Access
+## Create an account at Caltech T2 / iBanks GPU cluster
 
-The head node `ibanks.hep.caltech.edu` is running the nfs home and data server.
-It has a public (regular network) and a private (10G) IP.
+Either your postdoc contact at Caltech or the T2 admins (t2admin@hep.caltech.edu) can create an account for you. Send the following info to them:
 
-Worker nodes are a follows, in chronological order of creation
-* `culture-plate-sm.hep.caltech.edu` is a Supermicro server with 2T of local SSD, and runs 8 NVidia GeForce GTX 1080
-* `imperium-sm.hep.caltech.edu` is a Supermicro server with 2T of local SSD, and runs 8 NVidia GeForce GTX 1080
-* `flere-imsaho-sm.hep.caltech.edu` is a Supermicro server with 2T of local SSD, and runs 6 NVidia Titan Xp
-* `mawhrin-skel-sm.hep.caltech.edu` is a Supermicro server with 2T of local NVME running 2 NVidia GeForce GTX Titan X
+* your full name and institutional contact email
+* your CERN lxplus account name. If you are not a CERN user, give the access.caltech or other established institutional account name.
+* an SSH public key
+* initial account expiry date, which can be extended upon need (max 4 years from now for Caltech internal, max 1 year for external)
+* an X509 grid certificate distinguished name in the form of: /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=... from CRAB Prerequisites - Grid certificate or http://cilogon.org (search for your institute)
```

Comment:
not sure the X509 is an absolute necessity for gpuusers, also how does one get one if not by grid certificate?

Reply:
It is optional. Anyone can get it from cilogon.org ( Login with their institution or even google.). All transfer nodes trust CILogon Certificates, but only specific DN's. So if all or some GPU Users have a certificate - they can store files to Hadoop or access them via xrootd.